### DISCUSSION TITLE
"Classic" User Interface / Window

### DISCUSSION THREAD

**Discussion by OfflinePing** [AllenDang/giu#836] [Q&A] [answered]
Hey,
Just wanted to ask if it is possible to make the Window look like a classic inGUI Application (see screenshot below). Inclusive, the Border and so on.

![image](https://github.com/user-attachments/assets/474325cc-7e87-4c3d-87d6-6f2ee66610f1)

**Comment by gucio321** [comment]
Sure, you can use style setter or there was an issue for themes

**Reply by OfflinePing** [reply]
But is it actually possible to even design the Windows "default" bar where the close button is located?

**Reply by gucio321** [reply]
Honestly i was also wondering where is the close button 

I'll check it in a few days

**Comment by OfflinePing** [comment]
Okay, I saw the frameless flag. But is there a Widget or something to make it moveable manually?

**Reply by gucio321** [reply]
What do you mean by movable manually? 
You can use WindowWidget and it will be movable

**Reply by OfflinePing** [reply]
When you do a g.NewMasterWindow with a g.SingleWindow() inside it and set the g.MasterWindowFlagsFrameless flag you are not able to move the Window around

**Reply by OfflinePing** [reply]
so basically I am trying to make a WindowWidget without the MasterWindow in the background in terms of Look & Feel of my Application

**Reply by OfflinePing** [reply]
technically setting the Master Window to heigh 1x1px would work but it feels wrong doing xD

**Reply by gucio321** [reply, marked as answer]
Imo you should use windowwidget with transparent frameless master window
